hicommonwealth/commonwealth
# Updated reaction count state

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6772

## Description of Changes
- Updates reaction count of threads and comments based on user's vote weight, when user adds/removes reactions.

## "How We Fixed It"

By updating local state whenever user adds/removes recations

## Test Plan
- Buy stake in a community
- Add/Remove reaction to/from any thread, verify reaction count updates correctly based on your vote weight
- Add/Remove reaction to/from any thread's comment, verify reaction count updates correctly based on your vote weight

## Deployment Plan
N/A

## Other Considerations
N/A
